<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerbDialog</class>
 <widget class="QDialog" name="VerbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="imperativField"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="infinitivField"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="presensField"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="preteritumField"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="imperativLabel">
     <property name="text">
      <string>imperativ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="infinitivLabel">
     <property name="text">
      <string>infinitiv</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="presensLabel">
     <property name="text">
      <string>presens</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="preteritumLabel">
     <property name="text">
      <string>preteritum</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="supinumField"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="supinumLabel">
     <property name="text">
      <string>supinum</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="engTransLabel">
     <property name="text">
      <string>english translation</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="engTransField"/>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VerbDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VerbDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
